guides/v2.1/contributor-guide/templates/basic_template.md

```diff
@@ -14,12 +14,14 @@ You can start off by editing the local version of this file using markdown langu
 
 ## Metadata parameters
 
-Add the following {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters at the top of your page. We use this data when generating the DevDocs for the following:
+Add the following metadata parameters at the top of your page. We use this data when generating the DevDocs for the following:
 
 Parameter | Description
 --- | ---
-`group:` | The table of contents this file belongs to. Only add the name of the file without the extension. For example, the group of this file is `group: contributor-guide`.
+`group:` | The table of contents this file belongs to, which points to files located in `_data/toc/`. Only add the name of the file without the extension. For example, the group of this file is `group: contributor-guide`, which points to the table of contents defined in `_data/toc/contributor-guide.yml`.
 `title:` | The title of the page.
+`functional_areas:`  |  Optional. Adds facets for search results. Available facets include: sales, products, carts, customers, marketing, account, content, reports, stores, system, catalog, orders, frontend, theme, staging, search, configurations, integration, services, tools, setup, testing or test, standards, install, upgrade, B2B, cloud, and bundled extensions.
```

Comment:
The facets are case sensitive.
Update the list with the exact capitalization as defined in our internal wiki list.

Reply:
Added that info, and verified all entries match the list.